include/librealsense2/h/rs_sensor.h

```diff
@@ -440,6 +440,14 @@ void rs2_register_extrinsics(const rs2_stream_profile* from,
  */
 void rs2_get_video_stream_intrinsics(const rs2_stream_profile* mode, rs2_intrinsics* intrinsics, rs2_error** error);
 
+/**
+ * When called on a video profile, returns the depth offset
```

Comment:
Please specify the depth units and the orientation vector (sensor to optics or vice versa)